public/js/index.js

```diff
@@ -2360,22 +2389,63 @@ function initTopicbar() {
             throttle: 500,
             cache: false,
             onResponse: function(res) {
-                var formattedResponse = {
+                let formattedResponse = {
```

Comment:
can you keep `var`? The versions of IE we support don't support newer ECMAscript